Documentation/proposals/202411-zone-aware-sharding.md

```diff
@@ -0,0 +1,295 @@
+# Zone aware sharding for prometheus
```

Comment:
@simonpasquier do you think makes sense we highlght that additionalScrapeConfig will not automatically work?